lib/server.go

```diff
@@ -1822,6 +1856,49 @@ func (srv *Server) _addNewTxn(
 	return []*MsgDeSoTxn{txn}, nil
 }
 
+func (srv *Server) _addNewTxns(
+	pp *Peer, txns []*MsgDeSoTxn, rateLimit bool, verifySignatures bool) ([]*MsgDeSoTxn, error) {
```

Comment:
Just a nit lol. It's nice to place function params on a different line from return types when the function signature is long. Generally more convenient for future readers to eyeball the signature
```suggestion
func (srv *Server) _addNewTxns(
	pp *Peer, txns []*MsgDeSoTxn, rateLimit bool, verifySignatures bool,
) ([]*MsgDeSoTxn, error) {
```